PYME/Acquire/SpoolController.py

```diff
@@ -78,9 +78,10 @@ def __init__(self, scope, defDir=genHDFDataFilepath(), defSeries='%(day)d_%(mont
         
         #dateDict = {'username' : win32api.GetUserName(), 'day' : dtn.day, 'month' : dtn.month, 'year':dtn.year}
         
-        self._user_dir = None
         self._base_dir = nameUtils.get_local_data_directory()
-        self._subdir = nameUtils.get_spool_subdir()
+        self._dirname = os.sep.join([self._base_dir, ] + nameUtils.get_spool_subdir())
+        self._cluster_dirname = self.get_cluster_dirname(self._dirname)
```

Comment:
is there any reason to have this as seperate variable and not just as a `.cluster_dirname` property which computes on the fly?

Reply:
Can't remember, but might be I was trying not to change more than necessary since I know I'm on eggshells when I touch spoolcontroller and didn't want to set off the alarms. `get_cluster_dirname` could definitely be a `cluster_dirname` property, though `_cluster_dirname` might be preferable since `dirname` has some provenance.